coalib/bearlib/languages/documentation/DocumentationComment.py

```diff
@@ -2,28 +2,32 @@
 
 
 @generate_repr()
-@generate_eq("documentation", "language", "docstyle", "marker", "range")
+@generate_eq("documentation", "language", "docstyle",
+             "indent", "marker", "range")
 class DocumentationComment:
     """
     The DocumentationComment holds information about a documentation comment
     inside source-code, like position etc.
     """
 
-    def __init__(self, documentation, language,
-                 docstyle, marker=None, range=None):
+    def __init__(self, documentation, language, docstyle,
+                 indent=None, marker=None, range=None):
```

Comment:
As it's difficult to find a name for it, I can agree with that, though maybe you want to change
"*in* the first marker" to "*in front* of the first marker"? As *in* means more *inside* of the marker I think^^